kafka-connect/kafka-connect/src/test/java/org/apache/iceberg/connect/data/RecordConverterTest.java

```diff
@@ -206,11 +227,16 @@ public static void beforeAll() {
 
   @BeforeEach
   public void before() {
+    super.before();
+    ImmutableMap<String, String> mp = ImmutableMap.of("write.format.default", format);
+    System.out.println(format);
```

Comment:
please remove these sout

Reply:
Oops looks like I missed removing these lines, thanks for catching !